engine/helpers.go

```diff
@@ -963,3 +990,86 @@ func genCert(targetDir string) error {
 	log.Infof(log.Global, "gRPC TLS key.pem and cert.pem files written to %s\n", targetDir)
 	return nil
 }
+
+// NewSupportedExchangeByName helps create a new exchange to be loaded that is
+// supported by GCT. This function will return an error if the exchange is not
+// supported.
+func NewSupportedExchangeByName(name string) (exchange.IBotExchange, error) {
+	switch strings.ToLower(name) {
+	case "binanceus":
+		return new(binanceus.Binanceus), nil
+	case "binance":
+		return new(binance.Binance), nil
+	case "bitfinex":
+		return new(bitfinex.Bitfinex), nil
+	case "bitflyer":
+		return new(bitflyer.Bitflyer), nil
+	case "bithumb":
+		return new(bithumb.Bithumb), nil
+	case "bitmex":
+		return new(bitmex.Bitmex), nil
+	case "bitstamp":
+		return new(bitstamp.Bitstamp), nil
+	case "bittrex":
+		return new(bittrex.Bittrex), nil
+	case "btc markets":
+		return new(btcmarkets.BTCMarkets), nil
+	case "btse":
+		return new(btse.BTSE), nil
+	case "bybit":
+		return new(bybit.Bybit), nil
+	case "coinut":
+		return new(coinut.COINUT), nil
+	case "exmo":
+		return new(exmo.EXMO), nil
+	case "coinbasepro":
+		return new(coinbasepro.CoinbasePro), nil
+	case "gateio":
+		return new(gateio.Gateio), nil
+	case "gemini":
+		return new(gemini.Gemini), nil
+	case "hitbtc":
+		return new(hitbtc.HitBTC), nil
+	case "huobi":
+		return new(huobi.HUOBI), nil
+	case "itbit":
+		return new(itbit.ItBit), nil
+	case "kraken":
+		return new(kraken.Kraken), nil
+	case "kucoin":
+		return new(kucoin.Kucoin), nil
+	case "lbank":
+		return new(lbank.Lbank), nil
+	case "okcoin":
+		return new(okcoin.Okcoin), nil
+	case "okx":
+		return new(okx.Okx), nil
+	case "poloniex":
+		return new(poloniex.Poloniex), nil
+	case "yobit":
+		return new(yobit.Yobit), nil
+	case "zb":
+		return new(zb.ZB), nil
+	default:
+		return nil, fmt.Errorf("'%s', %w", name, ErrExchangeNotFound)
+	}
+}
+
+// GetDefaultExchangeByName returns a defaulted exchange by its name if it exists.
+// This will allocate a new exchange and setup the default config for it. This
+// will automatically fetch available pairs.
+func GetDefaultExchangeByName(ctx context.Context, name string) (exchange.IBotExchange, error) {
```

Comment:
`CreateDefaultExchangeByName`/`NewExchangeByNameWithDefaults` or something since you're not really getting